x-pack/solutions/security/plugins/session_view/public/components/session_view/index.tsx

```diff
@@ -220,6 +251,19 @@ export const SessionView = ({
     fetchAlertStatus[0] ?? ''
   );
 
+  /**
+   * This useEffect should only impact the SessionView component when displayed in the expandable flyout.
+   * The SessionView tree and its detailed panel are separated and this allows the detailed panel to reset the
+   * view of the tree from the preview panel.
+   */
+  useEffect(() => {
+    if (resetJumpToEntityId && resetJumpToCursor) {
```

Comment:
Do we really need this `useEffect`?  Can we guard setting this state
```suggestion
    if (resetJumpToEntityId && resetJumpToCursor && (resetJumpToEntityId !== jumpToEntityId || resetJumpToCursor !== jumpToCursor) {
```